addons/payment/wizards/payment_link_wizard_views.xml

```diff
@@ -5,10 +5,14 @@
         <field name="model">payment.link.wizard</field>
         <field name="arch" type="xml">
             <form string="Generate Payment Link">
+                <group attr="{'invisible':[('partner_email', '=', False)]}">
+                    <p class="text-center text-warning font-weight-bold">No email set on the partner. The payment may fails for some Payment Acquirers (Stripe, ...)</p>
```

Comment:
Let's be a bit nicer :) (also, looks fugly in community)
```suggestion
                    <p class="alert alert-warning font-weight-bold">This partner has no email, which may cause issues with some payment acquirers. Setting an email for this partner is advised.</p>
```
and move it to just after the link field, i think people will be more likely to read it closer to the link rather than at the top (and update translation)